content/02.intro.md

```diff
@@ -7,11 +7,11 @@ Automated algorithms that extract meaningful patterns could lead to actionable k
 
 The term _deep learning_ has come to refer to a collection of new techniques that, together, have demonstrated breakthrough gains over existing best-in-class machine learning algorithms across several fields.
 For example, over the past five years these methods have revolutionized image classification and speech recognition due to their flexibility and high accuracy [@doi:10.1038/nature14539].
-More recently, deep learning algorithms have shown promise in fields as diverse as high-energy physics [@doi:10.1038/ncomms5308], dermatology [@doi:10.1038/nature21056], and translation among written languages [@arxiv:1609.08144].
+More recently, deep learning algorithms have shown promise in fields as diverse as high-energy physics [@doi:10.1038/ncomms5308], computational chemistry [@doi:10.1002/jcc.24764], dermatology [@doi:10.1038/nature21056], and translation among written languages [@arxiv:1609.08144].
```

Comment:
👍

Reply:
👍 

Also noting that we cite this review later in the intro.